src/openrct2/world/LargeScenery.cpp

```diff
@@ -34,19 +34,19 @@ colour_t LargeSceneryElement::GetTertiaryColour() const
 
 void LargeSceneryElement::SetPrimaryColour(colour_t newColour)
 {
-    assert(newColour <= 31);
+    assert(newColour < PALETTE_TOTAL_OFFSETS);
```

Comment:
The max colour is `COLOUR_COUNT` no one should be setting anything higher than that.

Reply:
This is going by palette map index, where the usable colors are no longer contiguous (the internal palette maps are now in the middle of the list). Will create a function to validate whether the colour_t is in either of the valid ranges.

Reply:
It would make more sense to just make them contiguous. They already are in colour_t and it is expected that they are contiguous.